README.md

```diff
@@ -30,3 +30,13 @@ Some pre-requisites:
 
 
 All further guidelines can be found [here](https://cod-ed.github.io/simulate/contributing).
+
+
+
+## Our Contributors  !! ✨
+### Thanks go to these wonderfull peoples: ✨
```

Comment:
Remove this line (37) and the emoticon from the line 36

Reply:
Ok sure Sir.